src/test/recipe.spec.ts

```diff
@@ -0,0 +1,509 @@
+import {
+  chai,
+  expect,
+  app,
+  StatusCodes,
+  RecipeDocument,
+} from "./config/server.config";
+import {
+  recipes,
+  deleteAllRecipes,
+  addSomeRecipes,
+} from "./mocks/recipe.mocks";
+import { getToken } from "./mocks/user.mocks";
+
+let token = "Bearer ";
+let _id: string;
+let updateRecipeId: string;
+
+before("Add user and get token", async function () {
+  token += await getToken();
+  _id = await addSomeRecipes();
+});
+
+after("Delete all recipes", async function () {
+  await deleteAllRecipes();
+});
+
+describe("Recipe testing", function () {
+  it("should return token", function () {
+    expect(token).to.be.a("string");
+  });
+
+  describe("Creating recipes", function () {
+    it("should create recipe", async function () {
+      const response = await chai
+        .request(app)
+        .post("/recipe/")
+        .set("content-type", "application/json")
+        .set("Authorization", token)
+        .send({
+          ...recipes.recipe,
+        });
+      expect(response).to.have.status(StatusCodes.OK);
+      expect(response.body).to.not.be.null;
+      expect(response.body).to.include.keys([
```

Comment:
The test itself is ok, but it showed me something: there is no need of returning `__v` in response.